addons/rhs/ReadMe.md

```diff
@@ -8,7 +8,6 @@ Alle Anpassungen des RHS Mods
 ## Developer:
 - [shukari](https://github.com/shukari)
 - [Eric Ruhland](https://github.com/Er1807)
-- [Eron](https://github.com/E-for-Eron)
```

Comment:
Warum ist Eron rausgeflogen?

Reply:
@E-for-Eron ist unwichtig, der macht nur so bischen Paint Müll. Zudem versucht er sich mit bisschen Config hoch runter zu behaupten. Komplett überschätzt hier.


<details>
  <summary>unwichtige Infos</summary>
bereits Maintainer
</details>